readme.md

```diff
@@ -191,40 +222,44 @@ The first argument is where we indicate what type of event we are listening to.
 
 #### `handleClickEvent`
 
-The second argument is where we indicate what we want to happen once the event occurs. This is what is known as a **callback**.  
+The second argument is our function defined above. This is what is known as a **callback**.
 
-This might not be your first time hearing it and definitely won't be your last. A callback is a piece of executable code that is passed as an argument to other code, which is expected to invoke (or "call back") that executable code at some convenient time.
+This might not be your first time hearing it and definitely won't be your last.
+A callback is a function that is passed as an argument to another function, the function which received the passed function now has access to it and can invoke (or "call back") the passed function whenever it would like. We will explore this idea in more detail in a later lesson.
```

Comment:
This sentence is heavy with the word `function` and considering we're working with two of them here it gets muddy.

A callback is a function that is passed as an argument to another function`. The receiving function now has access to the callback (function)` and can invoke (or "call back") the passed function whenever it would like. We will explore this idea in more detail in a later lesson.